scripts/run_tests_functional.sh

```diff
@@ -27,7 +27,7 @@ yarn test_unit_no_watch
 display_result $? 4 "Front end unit tests"
 
 if [ -z "$EQ_FUNCTIONAL_TEST_SUITES" ]; then
-    export EQ_FUNCTIONAL_TEST_SUITES="core"
+    export EQ_FUNCTIONAL_TEST_SUITES="core,components,census,ukis"
```

Comment:
do we want census and ukis in this list?

Reply:
If we wanted to run all the tests for census or ukis only?

Reply:
`yarn test_functional -- --suite=census`??

Reply:
yarn test_functional -- --suite=ukis??

Reply:
`yarn test_functional -- --suite=census`??

Reply:
yeh

Reply:
yep

Reply:
upgrade your yarn version, wont need the first `--` then